### PR TITLE
fix: remove credentials headers

### LINKED ISSUE #3: Fix s3

### PR DESCRIPTION
**Pull request by Julien-Torrent**
close #3

### CODE REVIEW

File: src/fileServices/s3Service.ts [hunk 1]
```diff
@@ -103,10 +103,15 @@ export class S3Service implements FileService {
       // check whether file exists
       await this.getMetadata(filepath);
 
+      const param = {
+        Bucket: bucket,
+        Key: filepath,
+        Expires: 60,
+      }
+
       // Redirect to url, TODO: Change for something better
```
Comment (pyphilia) [inline src/fileServices/s3Service.ts]: Remove comment?
Can you explain why you add the header manually?

File: src/fileServices/s3Service.ts [hunk 2]
```diff
@@ -103,10 +103,15 @@ export class S3Service implements FileService {
       // check whether file exists
       await this.getMetadata(filepath);
 
+      const param = {
+        Bucket: bucket,
+        Key: filepath,
+        Expires: 60,
```
Comment (pyphilia) [inline src/fileServices/s3Service.ts]: Factor out.

**Review by pyphilia — CHANGES_REQUESTED** [review]
🪶

File: src/fileServices/s3Service.ts [hunk 3]
```diff
@@ -41,7 +42,7 @@ export class S3Service implements FileService {
     const { s3Bucket: bucket } = this.options;
 
     const params = {
-      CopySource: originalPath,
+      CopySource: `${bucket}/${originalPath}`,
       Bucket: bucket,
```
Comment (pyphilia) [inline src/fileServices/s3Service.ts:46]: I guess it was needed, but I don't get why we need the bucket parameter AND the bucket in the "key".

**Review by pyphilia — APPROVED** [review]

File: src/fileServices/s3Service.ts (hunk 3 above)
Comment (Julien-Torrent) [reply src/fileServices/s3Service.ts:46]: I think it's because you can copy from an other bucket

**Pull request merged**